### PR TITLE
increase test coverage

### PR DESCRIPTION
**Pull request by svetivanova**
tests for GlobalErrorHandlerService, SkillGroupsService, SkillUsersService, SkillGroupsNewComponent
@LebedkoDmitry could you please take a look?

### CODE REVIEW

File: src/app/error/global-error-handler.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    const errorEvent = new ErrorEvent('', {
+      message: message
+    });
+    const err: HttpErrorResponse = new HttpErrorResponse({
+      error: errorEvent
+    });
+    expect(globalErrorHandlerService.createFullMessage(err)).toBe(message);
+  });
+
+  it('should take message text from HttpErrorResponse.message in case of an error from other servers', () => {
```
Comment (LebedkoDmitry) [inline src/app/error/global-error-handler.service.spec.ts]: Is `HttpErrorResponse.message`a misprint? Should `HttpErrorResponse.statusText` be here?

File: src/app/skill-groups/skill-groups.service.spec.ts [hunk 2]
```diff
@@ -21,6 +23,157 @@ describe('SkillGroupsService', () => {
...
+  it('should send a request to check if the skill already exists', () => {
+    const searchParam = 'group';
+
+    skillGroupsService.isSkillGroupExist(searchParam).subscribe(actualResponse => {
+      expect(actualResponse).toBeTruthy();
+    });
+
+    const req = httpTestingController.expectOne( `${environment.serverApiUrl}/groups/group-existence?search=${searchParam}`);
+    expect(req.request.method).toBe('GET');
+    req.flush(1);
```
Comment (LebedkoDmitry) [inline src/app/skill-groups/skill-groups.service.spec.ts]: It is better to return `true` here instead of `1`.

Comment (svetivanova) [reply src/app/skill-groups/skill-groups.service.spec.ts]: I thought about it. but there is a bug in angular, so i found this [workaround ](https://github.com/angular/angular/issues/20690#issuecomment-403485479)with `req.flush(1)`

File: src/app/skill-groups/skill-groups.service.spec.ts [hunk 3]
```diff
@@ -21,6 +23,157 @@ describe('SkillGroupsService', () => {
...
+      expect(data).toBe(groupId);
+    });
+
+    const req = httpTestingController.expectOne({
+      method: 'DELETE',
+      url: `${environment.serverApiUrl}/groups/${groupId}`
+    });
+    expect(req.request.method).toBe('DELETE');
+
+    req.flush(groupId);
```
[the remark this answers is not in the document]
Comment (svetivanova) [reply src/app/skill-groups/skill-groups.service.spec.ts]: thank you. done

File: src/app/error/global-error-handler.service.spec.ts (hunk 1 above)
Comment (svetivanova) [reply src/app/error/global-error-handler.service.spec.ts]: thank you. done

**Pull request merged**